packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx

```diff
@@ -41,7 +41,7 @@ export const BottomPageSection: FC<IProps> = ({
   };
 
   return (
-    <div className={classes}>
```

Comment:
can you use the react-ui footer for this?

Reply:
diff style.